platforms/core-configuration/model-core/src/integTest/groovy/org/gradle/api/provider/CollectionPropertyIntegrationTest.groovy

```diff
@@ -153,13 +155,6 @@ task thing(type: SomeTask) {
 afterEvaluate {
     thing.prop = ["value 2"]
 }
-
-task before {
```

Comment:
This was not required for the test to pass, and is invalid code under CC: leads to error when serializing `before` due to the reference to `thing` (I believe).